packages/paste-website/src/components/callout/index.tsx

```diff
@@ -8,7 +8,7 @@ import {WarningIcon} from '@twilio-paste/icons/esm/WarningIcon';
 type CalloutVariants = 'info' | 'warning';
 
 interface CalloutTitleProps {
-  as: HeadingAsTags;
+  as?: HeadingAsTags;
```

Comment:
ℹ️  Type fix: This was previously typed as required, but there is a default value provided in the declaration.